congestion-control-charter.txt

```diff
@@ -26,14 +19,13 @@ protocols. The congestion control expertise in the working group would also
 make it a natural venue to take on other work related to indications of
 congestion, such as delay, and potentially queueing algorithms.
 
-Accordingly, CONGRESS is chartered to conduct a review of RFC 5033 and to
+CONGRESS is chartered to conduct a revisionof RFC 5033. The working group should
 consider a revision that relaxes requirements to encourage more experiments in
-the IRTF/IETF. Adoption of standards-track work should consider (1) empirical
-evidence of safety and (2) stated intent to deploy by major transport
+the IRTF/IETF. Adoption of standards-track congestion control algorithm should consider (1) empirical
+evidence of safety (for example - avoiding congestion collapse) and (2) stated intent to deploy by major transport

```

Comment:
Still unsure whether "safety" is the right word, but, at least, the example given is useful.